### DISCUSSION TITLE
A way to see commit's diff side-to-side when in Git bare repo?

### DISCUSSION THREAD

**Discussion by poige** [tpope/vim-fugitive#2048] [Q&A] [answered]
Everything I searched for seems to assume there's a checked out version underneath which isn't true for Git "bare" repos.

**Comment by tpope** [comment, marked as answer]
One way is to use `:Gedit commit:path/to/file` to open up a historical blob, and `:Gdiffsplit!` to diff against the version in the parent commit.
